### ISSUE TITLE
Process finished with exit code -1073740791 (0xC0000409)

### ISSUE THREAD

**Issue by pxpwoa** [catboost/catboost#1732]
Problem:
catboost version: 0.25.1
Operating System: windows10
GPU: geForce RTX 2060
CUDA: 11.3
```python
from catboost import CatBoostClassifier
train_data = [[0, 3], [4, 1],[8, 1],[9, 1]]
train_labels = [0, 0, 1, 1]
model = CatBoostClassifier(iterations=100,
                           task_type="GPU",
                           devices='0')
model.fit(train_data,
          train_labels,)
```
I just run the example code of training on GPU. Process finished with exit code -1073740791 (0xC0000409)  when training is over.
`98:	learn: 0.0508575	total: 202ms	remaining: 2.04ms`
`99:	learn: 0.0502894	total: 204ms	remaining: 0us`
`Process finished with exit code -1073740791 (0xC0000409)`

**Comment by kizill** [comment]
Can you check for this bug in new 0.26 version?

**Comment by pxpwoa** [comment]
> Can you check for this bug in new 0.26 version?

Update to 0.26 version，but the same error.

**Comment by kizill** [comment]
Fixed in #1763, will be in next release in a day or two.
